velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -669,6 +668,29 @@ class SharedArbitrationTest : public exec::test::HiveConnectorTestBase {
     return result;
   }
 
+  QueryTestResult runFakeTask(
```

Comment:
where is this used?

Reply:
I will use it in followup for fake memory task.